openmetadata-ui/src/main/resources/ui/src/pages/signup/index.test.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { act, fireEvent, render } from '@testing-library/react';
+import React, { ReactNode } from 'react';
+import Signup from '.';
+import * as AppState from '../../AppState';
+
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
+const mockAppState = AppState as any;
+
+jest.mock('react-router-dom', () => ({
+  useHistory: jest.fn(),
+}));
+
+jest.mock('../../components/containers/PageContainer', () => {
+  return jest
+    .fn()
+    .mockImplementation(({ children }: { children: ReactNode }) => (
+      <div data-testid="PageContainer">{children}</div>
+    ));
+});
+
+jest.mock('../../AppState', () => ({
+  __esModule: true,
+  default: null,
+}));
+
+jest.mock('../../axiosAPIs/miscAPI', () => ({
+  getLoggedInUserPermissions: jest.fn().mockResolvedValue({ data: {} }),
+}));
+
+jest.mock('../../axiosAPIs/userAPI', () => ({
+  createUser: jest.fn().mockResolvedValue({
+    data: {},
+  }),
+}));
+
+jest.mock('../../utils/UserDataUtils', () => ({
+  fetchAllUsers: jest.fn(),
+}));
+
+describe('Test for Signup page', () => {
```

Comment:
```suggestion
describe('Signup page', () => {
```

As this will be part of test only no need to mention `test`